amazon/src/shared/utils/validation/models/ValidatorFn.ts

```diff
@@ -0,0 +1 @@
+export type ValidatorFn = (text: string, options?: Object) => boolean;
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Consider using primitive 'object' type</b></div>
<div id="fix">

It's generally recommended to use the primitive `object` type instead of the upper-cased `Object` type. This change improves type safety and aligns with TypeScript best practices. Consider updating the type definition to use `object` instead of `Object`.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -1 +1 @@
 -export type ValidatorFn = (text: string, options?: Object) => boolean;
 +export type ValidatorFn = (text: string, options?: object) => boolean;
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/omkardongre/amazon-clone/pull/4#issuecomment-2466676739>#545432</a></i></small>
</div>

---
Is this a valid issue, or was it incorrectly flagged by the Agent?
- [ ] it was incorrectly flagged